clients/go/valkey-go.json

```diff
@@ -0,0 +1,18 @@
+{
+    "name": "valkey-go",
+    "description": "A fast Golang Valkey client that does auto pipelining and supports server-assisted client-side caching.",
+    "repo":"https://github.com/valkey-io/valkey-go",
+    "installation": "go get github.com/valkey-io/valkey-go",
+    "language":"go",
+    "package_size": "14.5M",
```

Comment:
I also think `package_size` should be removed. For Go, `package_size` doesn't seem very useful to users. The portion of the library in the final binary depends not only on the library version but also on how the user uses it and compiles the program.

Reply:
My concern is also package_size won't really be actively maintained at all, so it will also quickly fall out of date. I think we should omit it.